pawn/sourcemod/scripting/include/smmem/vec.inc

```diff
@@ -231,13 +229,6 @@ methodmap CUtlVector
 		return this.Find(val, size) >= 0;
 	}
 
-	public void Destruct(int elem, int size = 4)
-	{
-		// Okay so freeing random shit is bad, I think we can all agree on that
-		// Destruct calls the destructor which can be literally anything so do it yourself ig
-//		WriteVal(this.m_pMemory + elem * size, 0);
-	}
-
 	public void FastRemove(int elem, int size = 4)
 	{
 		this.Destruct(elem);
```

Comment:
I tried to compile this on SM1.11 and got the following errors. Can you please fix them before I go ahead and merge? Just need to remove the references on lines 234, 245 and 274-275.
```
C:\Users\johnm\Downloads\sourcemod-1.11.0-git6964-windows\addons\sourcemod\scripting\include\smmem\vec.inc(234) : error 105: cannot find method or property "CUtlVector.Destruct"
C:\Users\johnm\Downloads\sourcemod-1.11.0-git6964-windows\addons\sourcemod\scripting\include\smmem\vec.inc(245) : error 105: cannot find method or property "CUtlVector.Destruct"
C:\Users\johnm\Downloads\sourcemod-1.11.0-git6964-windows\addons\sourcemod\scripting\include\smmem\vec.inc(275) : error 105: cannot find method or property "CUtlVector.Destruct"
```